draft-salowey-wimse-arch.md

```diff
@@ -61,8 +61,23 @@ TODO Introduction
 
 # Security Considerations
 
-# Threat Model
-TODO Security
+## Threat Model
+
+### Traffic Interception
+
+Workloads communicating within an an applications may face different threats to traffic interception in different deployments. In many deployments security controls are deployed for internal communications at lower layers to reduce the risk of traffic observation and modification for network communications. When a security layer such as TLS is deployed in these environments the protection is usually hop-by-hop rather than end-to-end. Where TLS or other protection mechanisms terminate, information is left in the clear and information may be disclosed or modified.
```

Comment:
"hop-by-hop" and "end-to-end" mean wildly different things to different people.

I would suggest "When a security layer such as TLS is deployed in this environments in combination with TLS decryption middleware such as load balancers, gateways, proxies, and firewalls, the protection is provided only between each adjacent pair of TLS endpoints. There are no guarantees of correct identity passthrough in those middleware TLS interception devices and services."

Reply:
I made an attempt and incorporating this change.